pages/success.tsx

```diff
@@ -346,6 +350,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   const profile = {
     name: eventType.team?.name || eventType.users[0]?.name || null,
     theme: (!eventType.team?.name && eventType.users[0]?.theme) || null,
+    brandColor: eventType.team ? "#292929" : eventType.users[0].brandColor || "#292929",
```

Comment:
Let me see if I understand this logic, If it's a team event use our brand color. Else use the first user brand color, else our brand color. Why not just let it be undefined and let the default prop kick in?

```suggestion
    brandColor: eventType.team || eventType.users[0].brandColor,
```

Reply:
Yeah, I think this can be improved. But leaving it as `eventType.team` or `undefined` causes issues as `eventType.team` is an object. Defaulting to `null` in case of eventType.team does make sense though